PubMed.js

```diff
@@ -198,10 +198,16 @@ function detectWeb(doc, url) {
 
 	// determine if book or bookSection for PubMed Labs
 	var bookCitation = doc.getElementsByClassName('book-citation');
-	if (bookCitation.length > 0 && ZU.xpath(bookCitation, './/div[@class="affiliations"]')) {
+	if (bookCitation.length > 0 && ZU.xpath(doc, './/div[@class="affiliations"]')) {
 		// For a bookSection there are the affiliations of the authors of this
 		// section as well as the affiliations of the book authors.
-		var bookAffiliations = ZU.xpath(doc.getElementById('full-authors'), './/div[@class="affiliations"]/h3[@class="title"]').length > 1;
+		var bookAffiliations;
+		if (doc.getElementById('full-authors')) {
+			bookAffiliations = ZU.xpath(doc.getElementById('full-authors'), './/div[@class="affiliations"]/h3[@class="title"]').length > 1;
+		}
+		else {
+			bookAffiliations = ZU.xpath(doc, '//div[@id="full-view-heading"]//div[@class="affiliations"]//li').length > 1;
```

Comment:
No, this looks wrong. We want to distinguish whether there are two affiliation sections (one for the chapter authors and one for the book authors) but not whether there are multiple affiliations. Thus, the final `li` should be deleted.